main.py

```diff
@@ -1 +1,166 @@
-print("hello world ")
\ No newline at end of file
+import random
+
+EMPTY_CELL = "."
+FLOWER = "F"
+WEED = "W"
+
+
+def plant_random(garden, amount, item, row, col):
+    for i in range(amount):
+        while True:
+            coordi_flower_x = random.randint(0, row - 1)
+            coordi_flower_y = random.randint(0, col - 1)
+            if garden[coordi_flower_x][coordi_flower_y] == EMPTY_CELL:
+                garden[coordi_flower_x][coordi_flower_y] = item
+                break
+
+
+def create_garden(row, col):
+    """Return a Grid for the Garden with n by m squares and 4 flowers and 3 weedss ."""
+    garden = []
+
+    for r in range(row):
+        rows = []
```

Comment:
nit: this name is very misleading. This variable only holds 1 row, while the name suggests it has multiple.